README.md

```diff
@@ -33,7 +33,14 @@ python setup.py install
 
 ### Requirements
 
--   Python 2.7+ or Python 3.6+ (PyPy supported)
+- Python 3.6+ (PyPy supported)
+
+#### Python 2.7 deprecation
+
+[The Python Software Foundation (PSF)](https://www.python.org/psf-landing/) community [announced the end of support of Python 2](https://www.python.org/doc/sunset-python-2/) "as of 01 January 2020, no new bug reports, fixes, or changes will be made to Python 2, and Python 2 is no longer supported."
```

Comment:
```suggestion
[The Python Software Foundation (PSF)](https://www.python.org/psf-landing/) community [announced the end of support of Python 2](https://www.python.org/doc/sunset-python-2/) on 01 January 2020.
```
nit: The quote flows kind of strangely with the beginning of the sentence IMO. I think it's fine just to say they stopped supporting it on X date.